README.md

```diff
@@ -37,32 +37,36 @@ The major points are:
 1. Install GitLab.
  - https://about.gitlab.com/downloads/
 
-2. After GitLab is installed you may sign in with the `root` user and password `5iveL!fe`.
+1. After GitLab is installed you may sign in with the `root` user and password `5iveL!fe`.
```

Comment:
Why change these all to 1?  It makes this diff look a lot larger than I think it is.